rosdep/python.yaml

```diff
@@ -6553,6 +6553,7 @@ wxpython:
     artful: [python-wxgtk3.0]
     bionic: [python-wxgtk3.0]
     cosmic: [python-wxgtk3.0]
+    focal: [python3-wxgtk4.0]
```

Comment:
Mind making a new rosdep key for `python3-wxgtk4.0` instead of expanding the `wxpython` key? [REP 151](https://ros.org/reps/rep-0151.html#rosdep-keys) and the [contributing guide](https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md#python-rules) say to add new keys for Python 3 dependencies rather than expand old ones.

Reply:
Ah, I didn't see REP 151. I just crawled the file a bit and found similar examples like `python-pyudev` and `python-requests`. But I'll update according to the REP this afternoon!